lib/rules/disallow-comma-before-line-break.js

```diff
@@ -31,6 +36,30 @@
  *     two: 2
  * };
  * ```
+ *
+ * ##### Valid for `{"allExcept": ["function"]}`
+ *
+ * ```js
+ * var x = {
+ *     one: 1,
+ *     two: function() {}
+ * };
+ * ```
+ *
+ * ##### Valid for `{"lineBreak": true}`
```

Comment:
@mrjoelkemp,
Do you think this `lineBreak` could be a default behavior for multiline ObjectExpression? For example from current docs:

##### Valid
```js
var x = {
    one: 1
    , two: 2
};
```

Right now fixed variant of this rule is always single line: `{one: 1, two: 2}`